packages/mobile/package.json

```diff
@@ -25,6 +26,7 @@
         "react-native": "https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz",
         "react-native-screens": "2.10.1",
         "react-native-web": "0.13.9",
+        "react-responsive": "^8.1.0",
```

Comment:
When adding dependencies we typically try to pin them.
This is to maximize reproducibility. It's particularly useful when we can't know for sure that `semver` is respected.